maintainer-policies-and-guidelines.md

```diff
@@ -0,0 +1,23 @@
+# Maintainer Policies and Guidelines
+
+This document aims to clarify hard policies that Maintainers have to follow as well as less-enforced guidelines that are recommended to be followed. The goal is to help Maintainers understand what is expected of them and what actions should be done to reduce the chances of conflict within the Maintainer team.
+
+## Policies
+
+- `c` tags in 1.21+ NeoForge should be kept in sync with Fabric's `c` tags as close as possible. New `c` tags or changes to existing `c` tag should by ran by Fabric to ensure they will agree to make the same changes. `c` tags are to be a standard between loaders with Fabric being our main partner in this effort.
+
+  - Failure to meet this expectation will result in the change being undone so NeoForge matches Fabric again and the incident reviewed to understand how we failed to communicate with Fabric.
+
+## Guidelines
+
+- When making a "big-deal" PR, please post a link to the PR in the NeoForge Discord's maintenance-talk channel, ping the maintainer role, and leave a short sentence on what the PR is about.
+
+  - A "big-deal" PR is one that has a significant impact on many systems, changes the project ecosystem, or requires significant coordination with external projects.
+  
+    - An example of a significant impact on many systems, Transfer PR which involved creating a transaction system and refactoring the item/fluid/energy handlers. Many systems affected and massive amount of changes. This is a big deal.
```

Comment:
We had many discussions on this, both public and internal, and there is a dedicated brainstorming topic to which all maintainers got added. If anyone is not aware by now I don't know what to say.

Reply:
Transfer PR is just an example of a big deal PR Not an example of a miscommunication. Well, I guess the second Transfer PR being made could be seen as a miscommunication but I'm using Adrian's/Soaryn's PR here as one that is a Big Deal and other PRs of similar magnitude should have a Maintainer ping. 

Sorry if that wasn't clear. These are not examples of miscommunications. They are examples of big deal stuff